src/Compilers/Core/Portable/SynthesizedLocalKind.cs

```diff
@@ -232,6 +232,12 @@ internal enum SynthesizedLocalKind
         /// TODO: Avoid using lambdas and display classes for implementation of relaxation stubs and remove this kind.
         /// </summary>
         DelegateRelaxationReceiver = 0x101,
+
+        /// <summary>
+        /// The builder type of an interpolated string, when using the interpolated string builder pattern.
```

Comment:
>The builder type of an interpolated string, when using the interpolated string builder pattern. [](start = 12, length = 95)

The relationship between the builder type and the local is not clear. Is this statement talking about the type of the local?

Reply:
Rewording this for clarity.

---
In reply to: [610649620](https://github.com/dotnet/roslyn/pull/52422#discussion_r610649620) [](ancestors = 610649620)